examples/mongodbatlas_stream_privatelink_endpoint/confluent_serverless/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "confluent_environment" "staging" {
+  display_name = "Staging"
+}
+
+resource "confluent_private_link_attachment" "pla" {
```

Comment:
I prefer using `this` instead of `pla` when there is only a single resource of this type